### ISSUE TITLE
Can no longer move collection frame whit Soulshape Journal addon installed

### ISSUE THREAD

**Issue by martinboy1974** [Kiatra/BlizzMove#104]
### What game version did the bug occur in?

Retail

### Troubleshooting

- [X] I have verified that BlizzMove is up to date.
- [X] I have tried to reproduce the bug with no other addons loaded.
- [X] I have verified that the issue hasn't yet been reported.

### Describe the bug.

I get an error everytime I try to move the collection frame. I have turned off all addons except blizzmove and the rematch and soulshape jounal blizzmove plugin installed. Nothing wrong happens when I enable rematch addon, but when I enable soulshape journal addonm I get the error and the frame cant be moved.

### Steps to reproduce.

-

### Expected behavior.

-

### Errors

```lua
1x BlizzMove/BlizzMove.lua:412: BlizzMove: The frame you just moved (CollectionsJournal) is probably in a broken state, possibly because of other addons. Copy the text from the popup window, and report it to the addon author.
[string "=[C]"]: in function `error'
[string "@BlizzMove/BlizzMove.lua"]:412: in function <BlizzMove/BlizzMove.lua:386>
[string "@BlizzMove/BlizzMove.lua"]:649: in function <BlizzMove/BlizzMove.lua:632>
[string "=(tail call)"]: ?

Locals:
(*temporary) = "BlizzMove: The frame you just moved (CollectionsJournal) is probably in a broken state, possibly because of other addons. Copy the text from the popup window, and report it to the addon author."
```

**Comment by Numynum** [comment]
you should see a popup when you try to move the frame (besides the error you posted), can you copy/paste the message from that popup in here?

**Comment by martinboy1974** [comment]
This is the message that pops op in the frame:

BlizzMove: The frame you just moved (CollectionsJournal) is probably in a broken state, possibly because of other addons. Copy the text from this popup window, and report it to the addon author.

Bad anchor connections for "CollectionsJournal":

**Comment by Numynum** [comment]
and that's where it ends? nothing else in there? very odd..

**Comment by martinboy1974** [comment]
Yes, that and the first error. It only comes if I enable Soulshape Journal addon and try to move collection frame. And I do have your soulshape journal plug-in installed

**Comment by Numynum** [comment]
Is this happening on all characters? Or only a specific one?
I'm kinda grasping at straws here, since it works for me, and our debug tool isn't giving any relevant info :/

**Comment by martinboy1974** [comment]
It only happens on one character. I tried deleting the WTF folder, but still getting error and cant move collectionframe on that character. No issues with my other characters. After updating rematch addon today, I got a new error in your frame:

BlizzMove: The frame you just moved (CollectionsJournal) is probably in a broken state, possibly because of other addons. Copy the text from this popup window, and report it to the addon author.

Bad anchor connections for "CollectionsJournal":


"RematchPetPanel.List.ScrollFrame.ScrollBar.ScrollThumb" is outside anchor family, but referenced by "RematchPetPanel.List.ScrollFrame.ScrollBar.PageDownButton" (created in "Interface/AddOns/Rematch/Widgets/AutoScrollFrame.xml:27", and "Interface/AddOns/Rematch/Widgets/AutoScrollFrame.xml:141" respectively)

"RematchLoadoutPanel.TargetPanel.List.ScrollFrame.ScrollBar.ScrollThumb" is outside anchor family, but referenced by "RematchLoadoutPanel.TargetPanel.List.ScrollFrame.ScrollBar.PageUpButton" (created in "Interface/AddOns/Rematch/Widgets/AutoScrollFrame.xml:27", and "Interface/AddOns/Rematch/Widgets/AutoScrollFrame.xml:124" respectively)

"RematchOptionPanel.List.ScrollFrame.ScrollBar.ScrollThumb" is outside anchor family, but referenced by "RematchOptionPanel.List.ScrollFrame.ScrollBar.PageDownButton" (created in "Interface/AddOns/Rematch/Widgets/AutoScrollFrame.xml:27", and "Interface/AddOns/Rematch/Widgets/AutoScrollFrame.xml:141" respectively)

"RematchLoadoutPanel.TargetPanel.List.ScrollFrame.ScrollBar.ScrollThumb" is outside anchor family, but referenced by "RematchLoadoutPanel.TargetPanel.List.ScrollFrame.ScrollBar.PageDownButton" (created in "Interface/AddOns/Rematch/Widgets/AutoScrollFrame.xml:27", and "Interface/AddOns/Rematch/Widgets/AutoScrollFrame.xml:141" respectively)

"RematchOptionPanel.List.ScrollFrame.ScrollBar.ScrollThumb" is outside anchor family, but referenced by "RematchOptionPanel.List.ScrollFrame.ScrollBar.PageUpButton" (created in "Interface/AddOns/Rematch/Widgets/AutoScrollFrame.xml:27", and "Interface/AddOns/Rematch/Widgets/AutoScrollFrame.xml:124" respectively)

"RematchPetPanel.List.ScrollFrame.ScrollBar.ScrollThumb" is outside anchor family, but referenced by "RematchPetPanel.List.ScrollFrame.ScrollBar.PageUpButton" (created in "Interface/AddOns/Rematch/Widgets/AutoScrollFrame.xml:27", and "Interface/AddOns/Rematch/Widgets/AutoScrollFrame.xml:124" respectively)

**Comment by Numynum** [comment]
interesting ythat it's 1 specific character.. what class is it? (do you have another character of the same class where it does work?)

I'll check out the changes to rematch later, thanks for the info

**Comment by martinboy1974** [comment]
Its my hunter. I tested on another hunter. No issues there. All the same addons are enabled on all my characters.
